### DISCUSSION TITLE
Dropdown: Better Overview

### DISCUSSION THREAD

**Discussion by alexanderloewe** [craftcms/cms#13065] [Ideas] [answered]
Since Craft CMS 4.4 the dropdown fields have a new design, which looks visually better, but it is very difficult to get an overview of all options.

I would welcome it if the contents of the dropdown field as well as the multiple selection field were made clearer.

Setting the height of the field to 70vh would already help, as well as displaying the individual options somewhat more compactly/enlargely.

The subtitles should also be a bit more conspicuous in my eyes.

**Before**

<img width="360" alt="Bildschirmfoto 2023-04-08 um 14 31 09" src="https://user-images.githubusercontent.com/69263307/230721750-4ff26adb-6b33-4ef9-bebe-9ab25ea687a5.png">

**Now**

<img width="498" alt="Bildschirmfoto 2023-04-08 um 14 31 24" src="https://user-images.githubusercontent.com/69263307/230721656-c839965a-48ae-4d0b-ae09-522c34104c2e.png">

**Comment by mmikkel** [comment]
Selectize for custom Dropdown fields sounded like a good idea in theory, but in practice I think the AX improvement is questionable.

IMO, Selectize shines in cases where there are a ton of options, **and** the user has some idea up front of what those options are, so they're able to use the search to quickly find the one they're looking for. In pretty much all other cases (and I find this applies to most custom Dropdown fields) Selectize is arguably more cumbersome to use compared to native `<select>` inputs.

In my opinion, a new Dropdown field setting to enable/disable Selectize would probably be a good idea.

**Reply by thupsi** [reply]
I agree. I think this is an example where trying to replace a native feature you end up losing features and fail-safes you got for free, and run into all sorts of edge cases and stuff.

An option like @mmikkel suggests would be great!

**Reply by mmikkel** [reply]
I think Selectize is an awesome AX improvement in all cases where a native `<select>` input comes up short; e.g. if you need search/typeahead, custom option rendering, multi-select etc. It works great for a lot of the admin settings, for example.

IMO those extra features are all N/A for _most_ custom Dropdown fields though, and when it has no real use, Selectize certainly feels more clunky than a native select input.

**Comment by alexanderloewe** [comment]
I think in the Condition Dropdown it works fine:

<img width="962" alt="Bildschirm­foto 2023-04-15 um 16 55 51" src="https://user-images.githubusercontent.com/69263307/232232419-90bfcafd-ea3b-40c5-8617-63a8ffc0a91b.png">

**Reply by mmikkel** [reply]
That one is actually not a Selectize input; it’s a custom combobox component.

**Reply by alexanderloewe** [reply]
That's not my point. It's about showing that the design here is much clearer and user-friendly. 
You can adapt the design to the select field, that is what I'd like to show.

**Comment by brandonkelly** [comment, marked as answer]
We made some Selectize styling improvements for Craft 4.5 via #13176, and a couple additional improvements to address feedback in #13273.

**Reply by alexanderloewe** [reply]
Very good!

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out with Selectize improvements!

**Reply by alexanderbuergin** [reply]
Nice Update!
